app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -49,12 +54,17 @@ class StateFragmentPresenter @Inject constructor(
       it.stateFragment = fragment as StateFragment
       it.viewModel = getStateViewModel()
     }
-
+    explorationId = fragment.arguments!!.getString(EXPLORATION_ACTIVITY_TOPIC_ID_ARGUMENT_KEY)
+    getAudioFragment()
     subscribeToCurrentState()
 
     return binding.root
   }
 
+  private fun getAudioFragment(): AudioFragment? {
```

Comment:
I don't think this implementation is correct or maybe I am missing something here.

1. This is `getAudioFragment` which is getting used in `handlerCreateView` is not using the return value.
2. Also, this method is just returning the audio fragment but where is the code for showing that AudioFragment?

The audio fragment functionality is getting lost in this update.

Reply:
I missed the implementation. Added it now.